### ISSUE TITLE
[BUG] crc setup fails to check virtualization for Fedora arm64

### ISSUE THREAD

**Issue by adrianriobo** [crc-org/crc#4433]
### General information

 Pre testing of crc for arm64 on Fedora arm64

## Host Operating System
```bash
[fedora@ip-10-0-2-39 ~]$ cat /etc/*release
Fedora release 40 (Forty)
NAME="Fedora Linux"
VERSION="40 (Cloud Edition)"
ID=fedora
VERSION_ID=40
VERSION_CODENAME=""
PLATFORM_ID="platform:f40"
PRETTY_NAME="Fedora Linux 40 (Cloud Edition)"
ANSI_COLOR="0;38;2;60;110;180"
LOGO=fedora-logo-icon
CPE_NAME="cpe:/o:fedoraproject:fedora:40"
HOME_URL="https://fedoraproject.org/"
DOCUMENTATION_URL="https://docs.fedoraproject.org/en-US/fedora/f40/system-administrators-guide/"
SUPPORT_URL="https://ask.fedoraproject.org/"
BUG_REPORT_URL="https://bugzilla.redhat.com/"
REDHAT_BUGZILLA_PRODUCT="Fedora"
REDHAT_BUGZILLA_PRODUCT_VERSION=40
REDHAT_SUPPORT_PRODUCT="Fedora"
REDHAT_SUPPORT_PRODUCT_VERSION=40
SUPPORT_END=2025-05-13
VARIANT="Cloud Edition"
VARIANT_ID=cloud
Fedora release 40 (Forty)
Fedora release 40 (Forty)
[fedora@ip-10-0-2-39 ~]$ uname -a
Linux ip-10-0-2-39.ec2.internal 6.11.5-200.fc40.aarch64 #1 SMP PREEMPT_DYNAMIC Tue Oct 22 19:29:37 UTC 2024 aarch64 GNU/Linux
```

### Steps to reproduce

  1. Install crc linux arm64 distributable  
  2. Run crc setup - b bundle.arm64.crcbundle
  
### Expected

Setup ends up successfully 

### Actual

Setup ends up failing while checking if virtualization is enable

### Logs

```bash
....
INFO Checking if Virtualization is enabled        
DEBU Checking if the vmx/svm flags are present in /proc/cpuinfo 
DEBU Could not find cpu flags from /proc/cpuinfo  
INFO Setting up virtualization                    
You need to enable virtualization in BIOS
```

### Aditional information

Setting the config to skip the check for virtualization: `crc config set skip-check-virt-enabled true` setup ends up successfully and after run `crc start`  we have a running cluster:

```bash
[fedora@ip-10-0-2-39 ~]$ crc status
CRC VM:          Running
OpenShift:       Running (v4.17.1)
RAM Usage:       6.082GB of 10.93GB
Disk Usage:      20.58GB of 32.68GB (Inside the CRC VM)
Cache Usage:     19.53GB
Cache Directory: /home/fedora/.crc/cache

```

**Comment by rohanKanojia** [comment]
@adrianriobo : hello,  Is this issue fixed by #4437?

**Comment by adrianriobo** [comment]
Yeah indded,  thanks for the reminder

fixed by #4437
